src/index.html

```diff
@@ -3,11 +3,359 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link rel="icon" type="image/x-iconn"
+    href="images/logo/Air.png">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins:wght@600&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Dia</h1>
+
+    <header class="header">
+      <div class="header__icons">
+        <a href="#">
+          <img src="images/logo/Air.png" alt="logo">
+        </a>
+        <a href="#menu" class="icon icon--menu"></a>
+      </div>
+      <nav class="header__menu__nav">
+        <ul class="header__menu__nav--list">
+          <li><a href="#about-us" class="nav__link">About us</a></li>
```

Comment:
Links should have a smooth scroll for a better user experience [Checklist #8](https://github.com/mate-academy/layout_dia#:~:text=Add%20a%20smooth%20scroll%20for%20the%20whole%20page)